README.md

```diff
@@ -32,11 +32,12 @@ Once installed, you can send URLs to Kodi using one of the supported apps listed
 ### Mobile Apps
 - [Kore, Official Andorid Remote for Kodi](https://play.google.com/store/apps/details?id=org.xbmc.kore&hl=de&gl=US)
   - Check the app settings to always use sendtokodi if you get warnings about other missing kodi addons  
-- [Official SendToKodi iOS App](https://itunes.apple.com/de/app/sendtokodi/id1113517603?mt=8)
+- The **Official SendToKodi iOS App** has been retired for the time being, but if you have downloaded it before you should be able to do so again from your list of previously purchased apps in iOS App Store.
+- [Apple Shortcut](https://github.com/firsttris/plugin.video.sendtokodi/issues/104#issuecomment-2504081628) that shows up in the iOS share sheet when sharing a web address/link/URL. Download and open the .zip to install it to Apple's Shortcuts app, and then customize it with your details. Duplicate the shortcut for quick access to all your devices straight from the share sheet.
```

Comment:
can you push the zip file into the repo and link it in the readme?

Reply:
how big is it?

Reply:
I didn't feel that the source tree of the plugin was the right place for it, but sure, if you want it here. It might increase the threshold of updating it though, if people prefer to do their shortcut editing and uploading on mobile. But chances are it's already perfect 😇

Reply:
> How big is it?

26 kB, or 16 kB zipped

Reply:
The iOS app?! 😳

@anohren you [said](https://github.com/firsttris/plugin.video.sendtokodi/issues/104#issuecomment-2274192314) you have access to the original repo? Is it public? Could we have the app built there (via CI) or just release the artifact there somehow (and then link to it from here)?

Reply:
[Apple Shortcut](https://github.com/firsttris/plugin.video.sendtokodi/issues/104#issuecomment-2504081628)

Reply:
> you have access to the original repo? Is it public? Could we have the app built there

https://github.com/firsttris/plugin.video.sendtokodi/issues/104#issuecomment-2740504062

Reply:
@CasperMcFadden95 i unpublished this page, because i think its no longer valid, all current information are in the readme of the project. may i ask where do you got the link?

Reply:
or if you think the page is still needed, please elaborate why